extensions/blocks/revue/button-preview.js

```diff
@@ -0,0 +1,184 @@
+import { useCallback } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+const { getComputedStyle } = window;
+
+const isGradientAvailable = !! __experimentalUseGradient;
+
+const applyFallbackStyles = withFallbackStyles( ( node, ownProps ) => {
+	const { textColor, backgroundColor } = ownProps;
+	const backgroundColorValue = backgroundColor && backgroundColor.color;
+	const textColorValue = textColor && textColor.color;
+	//avoid the use of querySelector if textColor color is known and verify if node is available.
+	const textNode =
+		! textColorValue && node ? node.querySelector( '[contenteditable="true"]' ) : null;
+	return {
+		fallbackBackgroundColor:
+			backgroundColorValue || ! node ? undefined : getComputedStyle( node ).backgroundColor,
+		fallbackTextColor:
+			textColorValue || ! textNode ? undefined : getComputedStyle( textNode ).color,
+	};
+} );
+
+const MIN_BORDER_RADIUS_VALUE = 0;
+const MAX_BORDER_RADIUS_VALUE = 50;
+const INITIAL_BORDER_RADIUS_POSITION = 5;
+
+function BorderPanel( { borderRadius = '', setAttributes } ) {
+	const setBorderRadius = useCallback(
+		newBorderRadius => {
+			setAttributes( { borderRadius: newBorderRadius } );
+		},
+		[ setAttributes ]
+	);
+	return (
+		<PanelBody title={ __( 'Border settings', 'jetpack' ) }>
+			<RangeControl
+				value={ borderRadius }
+				label={ __( 'Border Radius', 'jetpack' ) }
+				min={ MIN_BORDER_RADIUS_VALUE }
+				max={ MAX_BORDER_RADIUS_VALUE }
+				initialPosition={ INITIAL_BORDER_RADIUS_POSITION }
+				allowReset
+				onChange={ setBorderRadius }
+			/>
+		</PanelBody>
+	);
+}
+
+function ButtonPreview( {
+	attributes,
+	backgroundColor,
+	className,
+	fallbackBackgroundColor,
+	fallbackTextColor,
+	setAttributes,
+	setBackgroundColor,
+	setTextColor,
+	textColor,
+} ) {
+	const { borderRadius, text } = attributes;
+
+	const { gradientClass, gradientValue, setGradient } = isGradientAvailable
+		? __experimentalUseGradient()
+		: {};
+
+	const classes = classnames( className, 'wp-block-button__link', {
+		'has-background': backgroundColor.color || gradientValue,
+		[ backgroundColor.class ]: ! gradientValue && backgroundColor.class,
+		'has-text-color': textColor.color,
+		[ textColor.class ]: textColor.class,
+		[ gradientClass ]: gradientClass,
+		'no-border-radius': borderRadius === 0,
+	} );
+
+	const styles = {
+		...( ! backgroundColor.color && gradientValue
+			? { background: gradientValue }
+			: { backgroundColor: backgroundColor.color } ),
+		color: textColor.color,
+		borderRadius: borderRadius ? borderRadius + 'px' : undefined,
+	};
+
+	return (
+		<>
+			<RichText
+				className={ classes }
+				onChange={ value => setAttributes( { text: value } ) }
+				placeholder={ __( 'Add text…', 'jetpack' ) }
+				style={ styles }
+				value={ text }
+				withoutInteractiveFormatting
+			/>
+
+			<InspectorControls>
+				{ isGradientAvailable && (
+					<PanelColorGradientSettings
+						title={ __( 'Background & Text Color', 'jetpack' ) }
```

Comment:
Same as comment above, would 'Button Background & Text Color' be background here as when panel open next to form it seems to indicate that the background and text colour of the whole form can be set:
<img width="820" alt="Screen Shot 2020-02-21 at 4 22 23 PM" src="https://user-images.githubusercontent.com/3629020/75001511-761b9b80-54c6-11ea-8fef-3dc6e93d1479.png">

Reply:
Fixed in 5e74e03a9c9d6bafc5c1e3d3a6c53258f9b3920d